config/sync/field.storage.taxonomy_term.computed_geolayer_field.yml

```diff
@@ -0,0 +1,51 @@
+uuid: cb283f7d-25e3-45a0-b362-0b966a5f25a0
+langcode: en
+status: true
+dependencies:
+  module:
+    - options
+    - taxonomy
+id: taxonomy_term.computed_geolayer_field
+field_name: computed_geolayer_field
+entity_type: taxonomy_term
+type: list_string
+settings:
+  allowed_values:
+    -
+      value: shoreline_length_computed
+      label: 'shoreline length computed'
```

Comment:
Let's make the labels 'sentence case' for example 'Shoreline length computed' with a capital 'S'